src/handler.py

```diff
@@ -214,12 +214,37 @@ def troubleshoot(event, context):
         _make_kms_key(event)
     elif event['action'].lower() == 'change_secret_kms_key':
         _change_secret_kms_key(event)
+    # TODO remove
+    elif event['action'].lower() == 'delete_stack':
```

Comment:
The development on geoserver sometimes corrupts the stack, so I need to delete it periodically.  I will add a TODO comment to remember to remove this.